From Daniel Dickman <daniel AT openbsd DOT org>

The Perl build is not deterministic. I'd like to propose committing this patch:

It is the same one found here:
https://github.com/Perl/perl5/commit/6baa8dbdfb8bc6c409778912cc3c8ea3dd39c342

Plus undefining PERL_BUILD_DATE to avoid adding it to the binary.

Index: perl.c
===================================================================
RCS file: /home/cvs/src/gnu/usr.bin/perl/perl.c,v
retrieving revision 1.25
diff -u -p -u -r1.25 perl.c
--- perl.c.orig	Sun Apr 10 17:49:39 2016
+++ perl.c	Sun Sep  4 05:04:49 2016
@@ -1835,6 +1835,8 @@
 #  endif
 #endif
 
+#undef PERL_BUILD_DATE
+
 #ifdef PERL_BUILD_DATE
     PUSHs(Perl_newSVpvn_flags(aTHX_
 			      STR_WITH_LEN("Compiled at " PERL_BUILD_DATE),
